.github/workflows/run-unit-tests.yml

```diff
@@ -124,7 +124,7 @@ permissions:
   contents: read
 jobs:
   tests:
-    timeout-minutes: 120
+    timeout-minutes: 35
```

Comment:
Maybe 60 instead?

Reply:
35 is fine. I looked at all the tests runs and we have 17-19 minutes max, so 35 is pretty high margin.